datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -845,7 +840,7 @@ pub struct SortExec {
     /// Fetch highest/lowest n results
     fetch: Option<usize>,
     /// Normalized common sort prefix between the input and the sort expressions (only used with fetch)
-    common_sort_prefix: LexOrdering,
+    common_sort_prefix: Vec<PhysicalSortExpr>,
```

Comment:
`ExternalSorter`'s `expr` is changed from `PhysicalSortExpr`s to `LexOrdering`, but this changes from `LexOrdering` to `PhysicalSortExpr`s. Can you clarify the reason? Why not having a consistent representation for sort expressions?

Reply:
Good observation -- this is because we don't have a proper ordering here, it is just a prefix of sort expressions (which can be empty unlike an ordering).